comp/host-profiler/collector/impl/otel_col_factories.go

```diff
@@ -14,6 +14,8 @@ import (
 	"github.com/DataDog/datadog-agent/comp/host-profiler/collector/impl/converternoagent"
 	"github.com/DataDog/datadog-agent/comp/host-profiler/collector/impl/receiver"
 	"github.com/DataDog/datadog-agent/comp/otelcol/otlp/components/processor/infraattributesprocessor"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/attributesprocessor"
```

Comment:
Looks like you are adding attributesprocessor too despite the title only mentions k8sattributesprocessor ?

Reply:
Indeed, I should update the PR description as I think adding `attributesprocessor` makes also sense.